src/components/hero/AcceptDemoInvitation.tsx

```diff
@@ -82,7 +82,7 @@ function AcceptDemoInvitation({
     };
 
     return (
-        <>
+        <Stack>
```

Comment:
Make sure the error shows vertically instead of horizontally

![image](https://github.com/estuary/ui/assets/270078/bb76ae3f-3b69-461b-b6ff-1fd75a123687)